.github/actions/copyAPIKeys/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -l
+
+JSON_FMT='{"google": {"client_id": "%s", "picker": "%s", "fonts": "%s"}}\n'
+
+printf "$JSON_FMT" "$CLIENT_ID" "$PICKER" "$PICKER" > .api_keys.dev.json
```

Comment:
I'm thinking just copy the example file here, the keys are never actually used, but the file needs to be present.
`cp example.api_keys.dev.json api_keys.dev.json`

Does that simplify other things?